src/library/scala/collection/mutable/OpenHashMap.scala

```diff
@@ -91,42 +104,43 @@ extends AbstractMap[Key, Value]
     table = new Array[Entry](newSize)
     mask = newSize - 1
     oldTable.foreach( entry =>
-      if (entry != null && entry.value != None) addEntry(entry))
+      if (entry != null && entry.value != None)
```

Comment:
But I do eliminate entries that have `value == None`, which, of course, eliminates the ones that also have `key == null.asInstanceOf[Key]`.